### PR TITLE
Add Hotcue Button colors to skins

### PR DESCRIPTION
**Pull request by Holzhaus**
This gives skins the ability to show Hotcue Button colors and adds support for that feature to the Deere skin. I used to colors from PR #2031 (and just added `0x20` to the dominant RGB channels for hover colors).

This resolves Launchpad Bug [lp1829884](https://bugs.launchpad.net/mixxx/+bug/1829884) (but only partially, since I only adapted the Deere skin).

Here's a screenshot:
Deere Hotcue Button colors

### CODE REVIEW

File: src/widget/controlwidgetconnection.cpp [hunk 1]
```diff
@@ -125,4 +126,6 @@ void ControlWidgetPropertyConnection::slotControlValueChanged(double v) {
         qDebug() << "Setting property" << m_propertyName
                 << "to widget failed. Value:" << parameter;
     }
+    pWidget->style()->unpolish(pWidget);
```
Comment (uklotzde) [inline src/widget/controlwidgetconnection.cpp]: A comment that explains why those 2 subsequent invocations are needed might help.

Comment (Holzhaus) [reply src/widget/controlwidgetconnection.cpp]: Added.

**Comment by Holzhaus** [comment]
I also added support for colored buttons to the other skins (except LateNight).

**Comment by Holzhaus** [comment]
@uklotzde IMHO this PR is ready. The only skin missing is the `LateNight` skin, but I have no idea how to add support to it, because it uses SVG files that contain gradients. In any case, this could be added to `LateNight` at a later date.

**Comment by ferranpujolcamins** [comment]
Good job, this PR looks good and it's a long awaited feature!

Can you tweak the default hotcue mark color on Tango and Deere so it matches the default color of the hotcue button?

**Comment by Holzhaus** [comment]
@ferranpujolcamins Done.

**Comment by ferranpujolcamins** [comment]
LGTM

**Comment by uklotzde** [comment]
LGTM. Very helpful, especially in conjunction with the auto-assignment of cue colors!

**Comment by ronso0** [comment]
LateNight is addressed in #2205

**Comment by ronso0** [comment]
@Holzhaus Could you explain which hotcue state [displayValue="2"] should address?
Different css colors for "1" and "2" don't yield any visible results, color from "1" is used for enabled and pressed buttons.
I'm trying to style both the `hotcue_X_enabled` and the `_activate` (set or play from hotcue) state.

**Comment by ronso0** [comment]
@Holzhaus 
In Shade (and only there) hotcue button #1 (and only this) never get's painted with the assigned color. Do you have any idea why this happpens?
The qss is correct, it doesn't matter at which position the marker is set, it doesn't matter if button#1 is the first hoctue template that's loaded...it's just weird.

**Comment by Holzhaus** [comment]
If the qss is okay, maybe the widget is not re-polished? But that shouldn't be skin-dependent...

**Pull request merged**